HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -4201,6 +4222,29 @@ def check_for_errors()
       errors << "RelatedHVACSystem '#{hvac_system.id}' is attached to multiple water heating systems."
     end
 
+    # Check for the sum of CFA served by distribution systems <= CFA
+    air_distributions = @hvac_distributions.select { |dist| dist if dist.distribution_system_type == HPXML::HVACDistributionTypeAir }
+    heating_dist = []
+    cooling_dist = []
+    air_distributions.each do |dist|
+      heating_systems = dist.hvac_systems.select { |sys| sys if (sys.respond_to? :fraction_heat_load_served) && (sys.fraction_heat_load_served > 0) }
+      cooling_systems = dist.hvac_systems.select { |sys| sys if (sys.respond_to? :fraction_cool_load_served) && (sys.fraction_cool_load_served > 0) }
+      if heating_systems.any? { |sys| sys.distribution_system_idref == dist.id }
```

Comment:
The ID check seems unnecessary. `dist.hvac_systems` will already only return HVAC systems that are attached to this distribution system.